.golangci.yml

```diff
@@ -60,7 +60,7 @@ linters-settings:
       - name: struct-tag
       # - name: string-format
       - name: string-of-int
-      - name: range-val-address
+      # - name: range-val-address
```

Comment:
Accidentally enabled before, I disabled it now until https://github.com/mgechev/revive/issues/553 is fixed